fe/src/components/user/donators.vue

```diff
@@ -1,32 +1,38 @@
 <template>
-  <table class="table-auto mt-6">
+  <table class="table-auto mt-6 w-full">
     <thead class="border-b border-primary-10">
       <tr class="h-10 items-center">
-        <th class="text-center px-4 text-primary-500">#</th>
+        <th class="text-center text-primary-500">#</th>
         <th class="text-left px-4 w-72">Donator</th>
         <th class="text-left px-4 flex-1">Campaign Title</th>
         <th class="text-left px-4 w-40">Amount</th>
       </tr>
     </thead>
     <tbody>
       <tr
-        v-for="({ id, avatar, donator, title, amount }, index) in donors"
-        :key="id"
+        v-for="(
+          { donator, campaignTitle, donationAmount }, index
+        ) in donatorsData.donatorsList"
+        :key="index"
```

Comment:
It is recommended to use `id` as :key instead of index - granted that the id is unique for every donor.
If so, kindly revert back to id. If it's not unique, keep this as is.
```suggestion
        :key="id"
```

Reply:
we dont have id, sir 🥲